Modix.Data/Models/Moderation/InfractionSummary.cs

```diff
@@ -116,12 +116,12 @@ public static ICollection<string> SortablePropertyNames
                 Duration = entity.Duration,
                 Subject = entity.Subject.Project(GuildUserBrief.FromEntityProjection),
                 CreateAction = entity.CreateAction.Project(ModerationActionBrief.FromEntityProjection),
-                RescindAction = (entity.RescindActionId == null)
+                RescindAction = (entity.RescindAction == null)
```

Comment:
This (and the rest) can now be further simplified to

```cs
entity.RescindAction?.Project(ModerationActionBrief.FromEntityProjection)
```

Yes?

Reply:
No. Expression trees cannot contain `?.`.